src/amo/components/CategoriesHeader/index.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import { categoriesFetch } from 'core/actions/categories';
+import translate from 'core/i18n/translate';
+import { getCategoryColor, visibleAddonType } from 'core/utils';
+import Button from 'ui/components/Button';
+import Card from 'ui/components/Card';
+import type { DispatchFunc } from 'core/types/redux';
+import LoadingText from 'ui/components/LoadingText';
+
+import './styles.scss';
+
+
+type CategoriesHeaderProps = {
+  addonType: string,
+  dispatch: DispatchFunc,
+  categories: Object,
+  clientApp: string,
+  error: boolean | null,
+  loading: boolean,
+  i18n: Object,
+}
+
+export class CategoriesHeaderBase extends React.Component {
```

Comment:
Make sure there's a bug on file to merge this with `src/amo/components/Categories`